attribution/experiment_logger.py

```diff
@@ -268,6 +299,7 @@ def print_tables(self):
         self.pretty_print(self.df_perturbations)
 
     def format_attribution_table(self):
+        # TODO: This is out of date, assuming df_token_attr should be df_token_attribution_matrix, "token_pos" needs to be updated to either input or output token pos.
```

Comment:
I've fixed it to work now as I think it was intended, but in general I think we should come back to these functions and refactor/remove some of them because it's hard to know what they are doing.